content/doc/book/using/referencing-another-project-by-name.adoc

```diff
@@ -0,0 +1,125 @@
+---
+layout: section
+title: Referencing another project by name
+---
+
+= Referencing another project by name
+
+In many places throughout Jenkins, you can refer to another project/job by name. For example, in a Pipeline Script, you might want to https://plugins.jenkins.io/copyartifact/[copy artifacts] from another project:
```

Comment:
Can use the `plugin` macro rather than the full URL to the plugin
```suggestion
In many places throughout Jenkins, you can refer to another project/job by name. For example, in a Pipeline Script, you might want to plugin:copyartifact/[copy artifacts] from another project:
```